### DISCUSSION TITLE
Spark provider not getting configured and visible in airflow web

### DISCUSSION THREAD

**Discussion by AkshayMehtaGithub** [apache/airflow#36657] [Q&A]
We are trying to setup apache airflow with spark on my local machine. We are using helm charts for deployment. We need to add spark connection in airflow web in order to be able to submit tasks to spark. But the spark connector details are not visible in the web. We have even added the install for apache-airflow-providers-apache-spark as per guideline. Kindly help if we are missing something. The airflow dockerfile is as below

```
FROM apache/airflow:2.6.2-python3.10
USER root
RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A8D3785C
RUN apt update
RUN apt install -y default-jdk
COPY ./spark /opt/spark

USER airflow
ENV JAVA_HOME='/usr/lib/jvm/java-11-openjdk-amd64'
ENV PATH="$PATH:/usr/lib/jvm/java-11-openjdk-amd64"
ENV PATH="$PATH:/opt/spark/bin:/opt/spark/sbin"
COPY ./spark /opt/spark
RUN pip install pyspark
RUN pip install apache-airflow-providers-apache-spark
ENV SPARK_HOME=/opt/spark

RUN pip install delta-spark
COPY ./jars /opt/spark/jars
```

**Comment by AkshayMehtaGithub** [comment]
Identified the mistake. Closing this. Thank you.

**Reply by potiuk** [reply]
What was it ? Maybe useful for others having the same feature @AkshayMehtaGithub ?

**Reply by AkshayMehtaGithub** [reply]
Ahh petty mistake. one of the team members mistakenly changed the image tag, to one which did not have the provider install. And no one realized it. And we were all the time thinking we are doing the install incorrectly
